### PR TITLE
Clean up bugs commented in the source code

### PR DESCRIPTION
**Pull request by vulcandth**
Please reference https://github.com/pret/pokecrystal/pull/912.

### CODE REVIEW

File: maps/Route15.asm [hunk 1]
```diff
@@ -193,6 +193,7 @@ SchoolboyBillyAfterBattleText:
 	done
 
 Route15SignText:
+; BUG: "Route 15" is not capitalized in a signpost (see docs/bugs_and_glitches.md)
 	text "Route 15" ; should be "ROUTE 15"
```
Comment (Rangi42) [inline maps/Route15.asm:197]:
```suggestion
	text "Route 15"
```

**Comment by Rangi42** [comment]
:question: Do you think the `BUG:` comments from pokecrystal should be in pokegold too, with the understanding that you'd use the pokecrystal bug docs for them? "see bugs_and_glitches.md" would lead you to see:

> All the bugs documented here were fixed in Pokémon Crystal. Any that weren't are already documented in [the pokecrystal repo](https://github.com/pret/pokecrystal/blob/master/docs/bugs_and_glitches.md), so they're not duplicated here.

**Comment by vulcandth** [comment]
> ❓ Do you think the `BUG:` comments from pokecrystal should be in pokegold too, with the understanding that you'd use the pokecrystal bug docs for them? "see bugs_and_glitches.md" leads you to see:
> 
> > All the bugs documented here were fixed in Pokémon Crystal. Any that weren't are already documented in [the pokecrystal repo](https://github.com/pret/pokecrystal/blob/master/docs/bugs_and_glitches.md), so they're not duplicated here.

I thought about it... I actually had the option to do that (I did a remote cerry-pick)... but then I removed them as I thought that might have been what you preferred.

**Comment by vulcandth** [comment]
It wouldn't be too difficult to re-add them. Let me know if you want them added or not. I'm indifferent on the matter.

**Comment by Rangi42** [comment]
> It wouldn't be too difficult to re-add them. Let me know if you want them added or not. I'm indifferent on the matter.

@mid-kid @aaaaaa123456789 @dannye Any opinions on the `BUG;` comments?

**Comment by aaaaaa123456789** [comment]
I see no good reason to not mark a bug simply because a separate project fixed it.

**Comment by vulcandth** [comment]
Okay I added the crystal bug comments back in.

**Review by Rangi42 — APPROVED** [review]

**Pull request merged**